test/SILGen/keypaths.swift

```diff
@@ -352,6 +385,19 @@ func subscripts<T: Hashable, U: Hashable>(x: T, y: U, s: String) {
 
   _ = \Subscripts<T>.[Bass()]
   _ = \Subscripts<T>.[Treble()]
+
+  _ = \SubscriptDefaults1.[]
+  _ = \SubscriptDefaults1.[0]
+  _ = \SubscriptDefaults1.[0, 0]
+  _ = \SubscriptDefaults1.[0, 0, 0]
+
+  _ = \SubscriptDefaults1.[false]
+  _ = \SubscriptDefaults1.[false, bool: false]
+  _ = \SubscriptDefaults1.[bool: false, 0]
+  _ = \SubscriptDefaults1.[bool: false, 0, 0]
+  
+  _ = \SubscriptDefaults2.[]
+  _ = \SubscriptDefaults3.[]
```

Comment:
Just for completeness can you add some examples where the arguments are fully specified as well?

Reply:
Good idea.